src/stores/crud-store.js

```diff
@@ -145,7 +145,8 @@ const configureStore = (options = {}) => {
         table: this.getInitialTableState(),
         query: this.getInitialQueryState(),
         isDataLake: false,
-        isReadonly: false
+        isReadonly: false,
+        status: 'initial'
```

Comment:
🤔 `status` was a bit ambiguous to me.  Maybe renaming to something like `hasAnyDocuments`, `isEmptyCursor`, `noResults` would fit the intent better?

Reply:
This 'initial' status indicates that we didn't fetch documents yet. I think your options sound more like we already tied to fetch something and received no results. we also use the same 'initial' status for handling zero state in explain plan plugin, therefore I used the same one here to be consistent. But i don't mind to change it if we will find the better name!